game-state-service

```diff
@@ -127,10 +149,18 @@ class GameStateService(Gio.Application):
         self._dbus_id = None
         self._state = None
 
+    def _emit_deletion_signals(self, key, connection, path, iface):
+        connection.emit_signal(None, path, iface, 'changed', GLib.Variant('(s)', (key,)))
```

Comment:
This signal requires a key and a value (different signature)

Reply:
Yes, I thought that would work, since I couldn't figure out how to pass a "null variant" like: `GLib.Variant('(sv)', (key, None))`, that didn't work.